Applications: nrf5340_audio: Changed buf counts (#22887)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

    The :ref:`net_buf_interface` will also contain the metadata about the audio stream in the ``user_data`` section of the API.
    This change was done to transition to standard Zephyr APIs, as well as to have a structured way to pass N-channel audio between modules.
  * The optional buildprog tool to use `nRF Util`_ instead of nrfjprog that has been deprecated.

>   * Buffer count (CONFIG_BT_ISO_TX_BUF_COUNT and CONFIG_BT_BUF_ACL_TX_COUNT) to be in-line with SoftDevice Controller (SDC) defaults. This can be changed and optimized for specific use-cases.

Reviewer: ```suggestion
  * The buffer count (CONFIG_BT_ISO_TX_BUF_COUNT and CONFIG_BT_BUF_ACL_TX_COUNT) to be in-line with SoftDevice Controller (SDC) defaults. This can be changed and optimized for specific use cases.
```

Reviewer 2: In addition to Greg's suggestion, move the second sentence to the new line:
"This can be changed and optimized for specific use cases."

Reviewer 2: It seems this suggestion is not implemented.

    This change was done to transition to standard Zephyr APIs, as well as to have a structured way to pass N-channel audio between modules.
  * The optional buildprog tool to use `nRF Util`_ instead of nrfjprog that has been deprecated.
  * The documentation pages with information about the :ref:`SD card playback module <nrf53_audio_app_overview_architecture_sd_card_playback>` and :ref:`how to enable it <nrf53_audio_app_configuration_sd_card_playback>`.

>   * Buffer count (CONFIG_BT_ISO_TX_BUF_COUNT and CONFIG_BT_BUF_ACL_TX_COUNT) to be in-line with SoftDevice Controller (SDC) defaults. This can be changed and optimized for specific use-cases.

Reviewer 2: ```suggestion
  * The buffer count (:kconfig:option:`CONFIG_BT_ISO_TX_BUF_COUNT` and :kconfig:option:`CONFIG_BT_BUF_ACL_TX_COUNT`) to be in-line with SoftDevice Controller (SDC) defaults.
    This can be changed and optimized for specific use-cases.
```

    This change was done to transition to standard Zephyr APIs, as well as to have a structured way to pass N-channel audio between modules.
  * The optional buildprog tool to use `nRF Util`_ instead of nrfjprog that has been deprecated.
  * The documentation pages with information about the :ref:`SD card playback module <nrf53_audio_app_overview_architecture_sd_card_playback>` and :ref:`how to enable it <nrf53_audio_app_configuration_sd_card_playback>`.
  * The buffer count (:kconfig:option:`CONFIG_BT_ISO_TX_BUF_COUNT` and :kconfig:option:`CONFIG_BT_BUF_ACL_TX_COUNT`) to be in-line with SoftDevice Controller (SDC) defaults.

>     This can be changed and optimized for specific use-cases.

Reviewer: ```suggestion
    This can be changed and optimized for specific use cases.
```

  * Updated the :ref:`fast_pair_locator_tag_testing_fw_update_notifications` section to improve the test procedure.
    The application provides now an additional log message to indicate that the firmware version is being read.

> Bluetooth nRF_Auraconfig sample
> -------------------------------
> 
>  * Buffer count (CONFIG_BT_ISO_TX_BUF_COUNT) to be in-line with SoftDevice Controller (SDC) defaults. This can be changed and optimized for specific use-cases.

Reviewer: Move under Bluetooth samples, as this sample does not have its own section under Samples.
After moving, change entry to following:
```suggestion
* :ref:`nrf_auraconfig` sample:

  * Updated the buffer count (:kconfig:option:`CONFIG_BT_ISO_TX_BUF_COUNT`) to be in-line with SoftDevice Controller (SDC) defaults. This can be changed and optimized for specific use cases.
```

Reviewer 2: In addition to Greg's suggestion, move the second sentence to the new line:
"This can be changed and optimized for specific use cases."

  * Added a workaround to an issue with unexpected disconnections that resulted from improper handling of the Bluetooth Link Layer procedures by the connected Bluetooth Central device.
    This resolves the :ref:`known issue <known_issues>` NCSDK-33632.

> * :ref:`nrf_auraconfig` sample:
>   * Updated the buffer count (:kconfig:option:`CONFIG_BT_ISO_TX_BUF_COUNT`) to be in-line with SoftDevice Controller (SDC) defaults.

Reviewer 2: ```suggestion
* :ref:`nrf_auraconfig` sample:

  * Updated the buffer count (:kconfig:option:`CONFIG_BT_ISO_TX_BUF_COUNT`) to be in-line with SoftDevice Controller (SDC) defaults.
```
